tests/cli/test_nodes.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2022 AI Singapore
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import io
+import math
+from pathlib import Path
+
+from click.testing import CliRunner
+
+from peekingduck.cli import cli
+from peekingduck.declarative_loader import PEEKINGDUCK_NODE_TYPES
+
+PKD_DIR = Path(__file__).resolve().parents[2] / "peekingduck"
+PKD_CONFIG_DIR = PKD_DIR / "configs"
+
+
+def available_nodes_msg(type_name=None):
+    def len_enumerate(item):
+        return int(math.log10(item[0] + 1)) + len(item[1])
```

Comment:
Related to above comment on `_len_enumerate()`.

Reply:
Moved `+ 1` to be outside `log10()`